setup.py

```diff
@@ -27,7 +27,7 @@
     package_data={"fourmat": ("assets/*.*", "assets/.*")},
     install_requires=(
         "click>=7",
-        "black==22.3.0",
+        "black==22.10.0",
```

Comment:
Should we still pin? I think the last time we updated this the answer was "yes".

Reply:
yeah, we should keep pinning